flatpak/org.dash.DashEvoTool.yml

```
@@ -0,0 +1,75 @@
+# Flatpak manifest for Dash Evo Tool
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)